plugin/my_asset_manager/MyAssetManagerInterface.py

```diff
@@ -61,24 +62,24 @@ def info(self):
         # to acquire the GIL and enter Python.
         return {constants.kInfoKey_EntityReferencesMatchPrefix: self.__reference_prefix}
 
-    def managementPolicy(self, traitSets, context, hostSession):
+    def managementPolicy(self, traitSets, policyAccess, context, hostSession):
         # The management policy defines which traits the manager is
-        # capable of imbuing queried traitSets with. In this case, we
-        # manage locations of assets only, and only in a read, not a
-        # write, context. Note `LocatableContentTrait` is a trait from
-        # the openassetio-mediacreation library, see :
+        # capable of imbuing queried traitSets with. In this case, the
+        # only workflow we support is to resolve the current location of
+        # pre-existing assets. Note `LocatableContentTrait` is a trait
+        # from the openassetio-mediacreation library, see :
         # https://github.com/OpenAssetIO/OpenAssetIO-MediaCreation
         policies = []
         for traitSet in traitSets:
             policy = TraitsData()
-            # The host asks specifically if a sets of traits are
-            # supported In this case, if any of the input traitSets are
+            # The host asks specifically if sets of traits are
+            # supported. In this case, if any of the input traitSets are
             # for read, and contain LocatableContent, as we can supply
             # data for that trait, we imbue a managed policy response,
             # as well as the traits we are able to supply data for. It's
             # important to get this right, for more info, see:
-            # https://openassetio.github.io/OpenAssetIO/classopenassetio_1_1v1_1_1host_api_1_1_manager.html#acdf7d0c3cef98cce7abaf8fb5f004354
-            if context.isForRead() and LocatableContentTrait.kId in traitSet:
+            # https://openassetio.github.io/OpenAssetIO/classopenassetio_1_1v1_1_1manager_api_1_1_manager_interface.html#ab86b5623a355d04086bae76875ebee17
```

Comment:
[Question] It's unfortunate that these hashes are based on the parameters of the method, as it makes linking to them unstable if we make changes. You think there's any way to link in a more stable manner? (I cant find anything.)
Good catch btw.

Reply:
I had a quick non-exhaustive google but didn't find a good solution :disappointed: